django/db/backends/postgresql/operations.py

```diff
@@ -382,11 +382,13 @@ def explain_query_prefix(self, format=None, **options):
             prefix += " (%s)" % ", ".join("%s %s" % i for i in extra.items())
         return prefix
 
-    def on_conflict_suffix_sql(self, fields, on_conflict, update_fields, unique_fields):
+    def on_conflict_suffix_sql(
+        self, fields, on_conflict, update_fields, unique_fields, where_clause=None
```

Comment:
Again, this `where_clause` argument should be propagated up to the parent in the super call below.

Also drop the `=None` as we'll always be passing something in explicitly.